style.css

```diff
@@ -0,0 +1,730 @@
+*,
+::before,
+::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'lato', sans-serif;
+}
+
+a {
+  text-decoration: none;
+}
+
+.black-bg {
+  background-color: black;
+}
+
+.black-list {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  width: 90%;
+  color: white;
+  list-style: none;
+  gap: 20px;
+  padding: 10px 50px;
+}
+
+.black-list a {
+  color: white;
+}
+
+.nav-container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  width: 75%;
+  margin: auto;
+}
+
+.brand-name {
+  color: black;
+  font-weight: bold;
+  font-size: 20px;
+}
+
+.brand-name a {
+  color: black;
+}
+
+.nav {
+  margin-top: 12px;
+}
+
+.nav-links {
+  display: flex;
+  list-style: none;
+  gap: 30px;
+}
+
+.nav-links a {
+  color: black;
+}
+
+.nav-item {
+  padding: 5px;
+  border: 3px solid white;
+}
+
+.nav-itemc {
+  border: 3px solid red;
+  color: #ff5438;
+  padding: 5px;
+}
+
+.nav-itemc a {
+  color: #ff5438;
+}
+
+/** CSS for Toggle menu **/
+
+.ham-icon {
+  display: block;
+  cursor: pointer;
+}
+
+.bars {
+  display: none;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background: black;
+}
+
+.hide {
+  visibility: hidden;
+}
+
+.banner-area {
+  background-image: url('images/banner.png');
+  background-repeat: no-repeat;
+  background-position: absolute;
+  background-size: cover;
+  width: 100%;
+}
+
+.banner-highlight {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  width: 73%;
+  margin: auto;
+  padding: 50px 70px 50px 0;
+}
+
+.initial-highlight {
+  font-size: 40px;
+  color: #ff5438;
+  margin-top: 70px;
+}
+
+.initial-highlight1 {
+  font-size: 40px;
+  color: #ff5438;
+  margin-top: 70px;
+}
+
+.heading {
+  font-size: 60px;
+  color: #ff5438;
+  font-weight: bold;
+}
+
+.highlight-detail {
+  background-color: #f9f7f7;
+  color: #515151;
+  border: 2px solid white;
+  width: 70%;
+}
+
+.highlight-detail p {
+  padding: 15px;
+}
+
+.time {
+  font-size: 25px;
+  font-weight: bold;
+}
+
+.location {
+  margin-bottom: 5%;
+  color: #67798e;
+  font-size: 20px;
+}
+
+.topic {
+  background-color: #282b32;
+  text-align: center;
+}
+
+.topic-area {
+  padding-top: 35px;
+}
```

Comment:
- [ ] As mentioned previously please make sure to add some vertical spacing between the cards in the "Main Topics" section in the mobile version as shown in the Behance design.
- Your design:
  ![image](https://github.com/nouman17610/developers-meetup/assets/104968236/df64641d-0b77-4bf5-a163-a9960b8253ee)
- Behance design: 
  ![](https://user-images.githubusercontent.com/93520178/257944932-aa39ec4c-f83f-4a15-8110-3fa18301b5ab.png)